mm2src/coins/utxo/utxo_common.rs

```diff
@@ -74,7 +74,7 @@ pub const DEFAULT_SWAP_VOUT: usize = 0;
 pub const DEFAULT_SWAP_VIN: usize = 0;
 const MIN_BTC_TRADING_VOL: &str = "0.00777";
 
-macro_rules! true_or {
+macro_rules! return_err_if_false {
```

Comment:
I think `true_or_err` is a better name

Reply:
but it actually does not create a 'true', that confused me at the first place.

Reply:
For me `true_or` sounded like Option's `ok_or` which returns Ok() or Err()

Reply:
hmm I think we can leave it as it is then